tests/ui/single_match_else.rs

```diff
@@ -32,4 +34,50 @@ macro_rules! unwrap_addr {
 
 fn main() {
     unwrap_addr!(ExprNode::Unicorns);
+
+    //
+    // don't lint single exprs/statements
+    //
+
+    // don't lint here
+    match Some(1) {
+        Some(a) => println!("${:?}", a),
+        None => return,
+    }
+
+    // don't lint here
+    match Some(1) {
+        Some(a) => println!("${:?}", a),
+        None => return,
+    }
```

Comment:
is this supposed to be the same?

Reply:
ah my bad, this is supposed to be the `{return}` case

Reply:
I know now, forgot to skip rustfmt on those cases.